consul/resource_consul_config_entry_ce_test.go

```diff
@@ -595,6 +595,8 @@ resource "consul_config_entry" "service_intentions" {
 	name = consul_config_entry.sd.name
 	kind = "service-intentions"
 
+	depends_on = [consul_config_entry.jwt_provider]
```

Comment:
This keeps tf from trying to delete the jwt provider before the service intention has been deleted. Before, the tf destroy was failing because Consul won't allow the provider to be removed if any intentions still reference it.